plugins/count.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: count
+spec:
+  version: v0.2.5
+  homepage: https://github.com/chenjiandongx/kubectl-count
+  shortDescription: Show resources count in the cluster.
```

Comment:
Nit: remove trailing dot

Nit: resources count → resource counts

I think you can rewrite this as

> Count resources by kind

and be more descriptive.

Reply:
short description has been changed as `Count resources by kind`